### PR TITLE
MUI `5.9.2` [generated]

### PR DESCRIPTION
**Pull request by aerialist7**

### CODE REVIEW

File: kotlin-mui/src/jsMain/generated/mui/base/OptionUnstyled.types.kt [hunk 1]
```diff
@@ -57,3 +54,7 @@ external interface OptionUnstyledProps<TValue> :
      */
     var label: String?
 }
+
+external interface OptionUnstyledType {
```
Comment (turansky) [inline kotlin-mui/src/jsMain/generated/mui/base/OptionUnstyled.types.kt:58]: Looks like redundant

File: kotlin-mui/src/jsMain/generated/mui/base/SelectUnstyled.types.kt [hunk 2]
```diff
@@ -52,56 +102,14 @@ external interface SelectUnstyledProps<TValue> : react.Props {
...
-     * @default undefined
-     */
-    var listboxOpen: Boolean?
-
-    /**
-     * Callback fired when the component requests to be opened.
-     * Use in controlled mode (see listboxOpen).
-     */
-    var onListboxOpenChange: ((isOpen: Boolean) -> Unit)?
+external interface SelectUnstyledType {
```
Comment (turansky) [inline kotlin-mui/src/jsMain/generated/mui/base/SelectUnstyled.types.kt:105]: Looks like redundant

**Review by turansky — CHANGES_REQUESTED** [review]

**Comment by lucgirardin** [comment]
Why are upgrading to MUI 5.9.2 (released in July 2023) and not to (one of) the latest version of MUI?!?

File: kotlin-mui/src/jsMain/generated/mui/base/OptionUnstyled.types.kt (hunk 1 above)
Comment (aerialist7) [reply kotlin-mui/src/jsMain/generated/mui/base/OptionUnstyled.types.kt]: Fixed

File: kotlin-mui/src/jsMain/generated/mui/base/SelectUnstyled.types.kt (hunk 2 above)
Comment (aerialist7) [reply kotlin-mui/src/jsMain/generated/mui/base/SelectUnstyled.types.kt]: Fixed

**Review by turansky — APPROVED** [review]

**Comment by aerialist7** [comment]
> Why are upgrading to MUI 5.9.2 (released in July 2023) and not to (one of) the latest version of MUI?!?

Hi @lucgirardin, please be patient and wait for updates - work in progress.

**Comment by lucgirardin** [comment]
> > Why are upgrading to MUI 5.9.2 (released in July 2022) and not to (one of) the latest version of MUI?!?
> 
> Hi @lucgirardin, please wait for updates - work in progress

Will exercise patience ;-)

**Pull request merged**